src/mistral_common/tokens/instruct/request.py

```diff
@@ -46,6 +47,16 @@ class InstructRequest(MistralBase, Generic[ChatMessageType, ToolType]):
     system_prompt: Optional[str] = None
     available_tools: Optional[List[ToolType]] = None
     truncate_at_max_tokens: Optional[int] = None
+    continue_final_message: bool = False
+
+    def model_post_init(self, context: Any) -> None:
+        super().model_post_init(context)
+
+        if self.continue_final_message:
+            if len(self.messages) == 0 or not isinstance(self.messages[-1], AssistantMessage):
+                raise ValueError("Cannot continue final message if the last message is not an assistant message.")
+            self.messages[-1].prefix = True
```

Comment:
Hmm don't like automatically overwriting attributes. Let's just throw an error if prefix is not True